1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

In [43]:
a = 1-0.95
M = 80
n = 256
std = 16
std_X = 16/n**0.5
Z = 1.96

In [44]:
M + Z*std_X

81.96

In [45]:
M - Z*std_X

78.04

Доверительный интервал [78.04, 81.96]

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,оценить истинное значение величины X при помощи доверительного интервала, покрывающего этозначение с доверительной вероятностью 0,95.


In [46]:
import numpy as np
X = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
n = 10
a = 1-0.95
v = n-1
t = 2.262
M = np.mean (X)
M


6.590000000000001

In [47]:
std_X= np.std(X, ddof= 1)
std_X

0.4508017549014448

In [48]:
M + t*std_X/n**0.5

6.912461744087575

In [49]:
M - t*std_X/n**0.5

6.267538255912426

Доверительный интервал [6.268, 6.912]

3,4 задачи решать через тестирование гипотезы

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметроказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.



a) H0: диаметр = 17 mm, mu = mu0

   H1: диаметр = 17.5 mm, mu > mu0

b) Известна сигма, берем критерий Z 

$\sigma  = std / \sqrt{n}$ 

In [54]:
n = 100
X_std = 4**(0.5)/n**0.5
X_std

0.2

In [55]:
Zh = (17.5 - 17) / X_std
Zh

2.5

Zh = 2.5

Z (5%) = 1.645

Zh > Z, значит H1 верна при α=0,05

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [56]:

X = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
M_X = np.mean (X)
M_X

198.5

H0: cредний вес = 200, mu = mu0

H1: cредний вес = 198.5, mu < mu0

In [57]:
a = 1-0.99
n = 10
v = n - 1
std_X = np.std (X, ddof = 1)
std_X 

4.453463071962462

In [58]:
th = (M_X - 200) / (std_X/n**0.5)
th

-1.0651074037450896

th = 1.0651

v = 9

a = 1

t (0.005) = 3.25

-t < th < +t, H0 верна при α=0,01